lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [{
+      type: 'object',
+      properties: {
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'object',
+          properties: {
+            'max': {
+              type: 'number',
+              minimum: 1
+            },
+            'allowFirstLine': {
+              type: 'boolean'
+            }
+          }
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0] || { singleline: 3, multiline: { max: 1, allowFirstLine: false }}
```

Comment:
I think that by having only one way of setting a rule we make user's lives easier, but I can change it if you don't agree :)

What do you think @michalsnik ?